journal/modernized/06-chapter-6.adoc

```diff
@@ -346,7 +361,11 @@ to our meeting-house in High-street, on the 24th of the third month, 1768,
 and after a solemn meeting, in which several living testimonies were borne,
 was interred in our burial ground in this city.
 
+[.signed-section-context-close]
```

Comment:
technically, this is more of a [signed-section-closing]... and since it is longer than one line, we would just have it be regular text. BUT.... I personally think it looks better the way that you have it here. So... why don't you just leave it, just this once :)